Week1/homework/js-exercises/arrayOfAnimals.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+//declare array variable
+let animals=[];
+
+//value of array
+console.log('value of array can contain list of objects');
+
+// print the array 
+console.log(animals)
+;
+//declare array variable
+let favoriateAnimals=['Horse', 'Cat', 'Dog'];
+
+//printing array
+console.log(favoriateAnimals);
+
+// push a new value to favoriate Animals array
+favoriateAnimals.push('Piglet');
```

Comment:
Well done using the `push` method / function! Great job!

Reply:
trying to do me best to keep learning